src/ui/components/pages/Assets.tsx

```diff
@@ -72,20 +72,26 @@ export function Assets({ setTab }: Props) {
     return <Intro />;
   }
 
-  const assetInfo = assets['WAVES'];
-
-  let wavesBalance;
-  if (assetInfo) {
-    const asset = new Asset(assetInfo);
-    wavesBalance = new Money(balances[address]?.available || 0, asset);
-  }
+  const amountInUsd = balances[address]?.assets
+    ? Object.entries(balances[address].assets).reduce(
+        (acc, [id, { balance }]) => {
+          if (assets[id]?.usdPrice) {
+            const tokens = new Money(balance, new Asset(assets[id])).toTokens();
+            acc += +assets[id].usdPrice * +tokens;
```

Comment:
If you use `getTokens` instead of `toTokens`, you'll get `BigNumber`, and then `acc` calculation would look like this:

```typescript
acc.plus(new BigNumber(assets[id].usdPrice).mul(tokens));
```

And `reduce` starting value would be `new BigNumber(0)`